api/src/test/java/com/github/commonsrdf/api/AbstractRDFTermFactoryTest.java

```diff
@@ -392,9 +363,9 @@ public void invalidLiteralLang() throws Exception {
 
 	@Test(expected = Exception.class)
 	public void invalidTriplePredicate() {
-		BlankNode subject = factory.createBlankNode("b1");
-		BlankNode predicate = factory.createBlankNode("b2");
-		BlankNode object = factory.createBlankNode("b3");
+		BlankNode subject = factory.createBlankNode();
+		BlankNode predicate = factory.createBlankNode();
+		BlankNode object = factory.createBlankNode();
 		factory.createTriple(subject, (IRI) predicate, object);
```

Comment:
I don't know, I only updated the factory/constructor parameters for this
patch, I didn't write any new tests. (Perhaps I should..)

Could you raise this separately against the master branch?
On 6 Feb 2015 01:10, "Peter Ansell" <notifications@github.com> wrote:

> In
> api/src/test/java/com/github/commonsrdf/api/AbstractRDFTermFactoryTest.java
> <https://github.com/commons-rdf/commons-rdf/pull/63#discussion_r24215971>:
>
> > @@ -392,9 +363,9 @@ public void invalidLiteralLang() throws Exception {
> >
> >  	@Test(expected = Exception.class)
> >  	public void invalidTriplePredicate() {
> > -		BlankNode subject = factory.createBlankNode("b1");
> > -		BlankNode predicate = factory.createBlankNode("b2");
> > -		BlankNode object = factory.createBlankNode("b3");
> > +		BlankNode subject = factory.createBlankNode();
> > +		BlankNode predicate = factory.createBlankNode();
> > +		BlankNode object = factory.createBlankNode();
> >  		factory.createTriple(subject, (IRI) predicate, object);
>
> Is this just testing for a ClassCastException? The use of
> expected=Exception.class is not recommended for that reason that you are
> not sure where the exception comes from (and using
> expected=SpecificException.class you are still not sure where the exception
> comes from)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/commons-rdf/commons-rdf/pull/63/files#r24215971>.
>